### ISSUE TITLE
/pytorch/c10/core/TensorImpl.h:1156.

### ISSUE THREAD

**Issue by the-cat-crying** [Megvii-BaseDetection/YOLOX#69]
~/mount_sdc/work_zp/Train/yolox$ python ./tools/demo.py video -n yolox-x -c /home/gy/mount_sdc/work_zp/Train/yolox/yolox_x.pth.tar --camid 0 --conf 0.3 --nms 0.65
2021-07-22 15:49:18 | INFO     | __main__:219 - Args: Namespace(camid=0, ckpt='/home/gy/mount_sdc/work_zp/Train/yolox/yolox_x.pth.tar', conf=0.3, demo='video', exp_file=None, experiment_name='yolox_x', fp16=False, fuse=False, name='yolox-x', nms=0.65, path='./assets/dog.jpg', save_result=False, trt=False, tsize=None)
/home/gy/Applications/anaconda3/envs/yolox/lib/python3.6/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
2021-07-22 15:49:20 | INFO     | __main__:229 - Model Summary: Params: 99.07M, Gflops: 281.93
2021-07-22 15:49:22 | INFO     | __main__:240 - loading checkpoint
2021-07-22 15:49:22 | INFO     | __main__:245 - loaded checkpoint done.

**Comment by ruinmessi** [comment]
This is a warning but not an error. Your code looks to run well. Just remember to add --save_result to save visual results

**Comment by Joker316701882** [comment]
#77 
This is a warning but no an error. But you can try our latest branch.
